sdk/tables/azure-data-tables/src/main/java/com/azure/data/tables/TableServiceClient.java

```diff
@@ -49,26 +49,180 @@
  * invoke REST API operations to make the requests and obtain the results that are returned.</p>
  *
- * <p>Instances of this client are obtained by calling the {@link TableServiceClientBuilder#buildClient()} method on a
- * {@link TableServiceClientBuilder} object.</p>
+ * <h2>Getting Started</h2>
+ *
+ * <p>The building and authenticating of instances of this client are handled by {@link TableServiceClientBuilder} instances. The following
+ * sample shows how to authenticate and build a TableServiceClient using a connection string.</p>
  *
- * <p><strong>Samples to construct a sync client</strong></p>
- * <!-- src_embed com.azure.data.tables.tableServiceClient.instantiation -->
+ * <!-- src_embed com.azure.data.tables.tableServiceClient.connectionstring.instantiation -->
  * <pre>
  * TableServiceClient tableServiceClient = new TableServiceClientBuilder&#40;&#41;
- *     .endpoint&#40;&quot;https:&#47;&#47;myvault.azure.net&#47;&quot;&#41;
- *     .credential&#40;new AzureNamedKeyCredential&#40;&quot;name&quot;, &quot;key&quot;&#41;&#41;
+ *     .connectionString&#40;&quot;connectionstring&quot;&#41;
  *     .buildClient&#40;&#41;;
  * </pre>
- * <!-- end com.azure.data.tables.tableServiceClient.instantiation -->
+ * <!-- end com.azure.data.tables.tableServiceClient.connectionstring.instantiation -->
+ *
+ * <p>See {@link TableServiceClientBuilder} documentation for more information on constructing and authenticating a client.</p>
+ *
+ * <p>The following samples show the various ways you can interact with the tables service using this client.</p>
+ *
+ * <hr/>
+ *
+ *
+ *
+ * <h3>Create a Table</h3>
+ *
+ * <p>The {@link #createTable(String) createTable} method can be used to create a new table within an Azure Storage or Azure Cosmos account.
+ * It returns a TableClient for the newly created table.</p>
+ *
+ * <p>The following sample creates a table with the name "myTable".</p>
+ *
+ * <!-- src_embed com.azure.data.tables.tableServiceClient.createTable#String -->
+ * <pre>
+ * TableClient tableClient = tableServiceClient.createTable&#40;&quot;myTable&quot;&#41;;
+ *
+ * System.out.printf&#40;&quot;Table with name '%s' was created.&quot;, tableClient.getTableName&#40;&#41;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.tables.tableServiceClient.createTable#String -->
+ *
+ * <hr/>
+ *
+ * <h3>Delete a Table</h3>
+ *
+ * <p>The {@link #deleteTable(String) deleteTable} method can be used to delete a table within an Azure Storage or Azure Cosmos account.</p>
+ *
+ * <p>The following sample deletes the table with the name "myTable".</p>
+ *
+ * <!-- src_embed com.azure.data.tables.tableServiceClient.deleteTable#String -->
+ * <pre>
+ * tableServiceClient.deleteTable&#40;&quot;myTable&quot;&#41;;
+ *
+ * System.out.printf&#40;&quot;Table with name '%s' was deleted.&quot;, &quot;myTable&quot;&#41;;
+ * </pre>
+ * <!-- end com.azure.data.tables.tableServiceClient.deleteTable#String -->
+ *
```

Comment:
Add here and for other samples,
 * <p><strong>Note:</strong> for asynchronous sample, refer to {@link async client}</p>